state-chain/runtime/src/lib.rs

```diff
@@ -421,7 +418,7 @@ impl pallet_cf_ingress_egress::Config<Instance2> for Runtime {
 	type AssetConverter = Swapping;
 	type FeePayment = Flip;
 	type SwapRequestHandler = Swapping;
-	type CcmAuxDataWitnessingHandler = SolanaAltWitnessingHandler;
+	type CcmAuxDataWitnessingHandler = CfSolanaAltWitnessingHandler;
```

Comment:
👀  - this is strange having solana in non-solana instances, my comment above would resolve this (by having an instance for each pallet - where all but solana would be noops).

Reply:
Yeah I've refactored this. Now only the Solana uses the Solana ALT handler, while the rest uses a "noop" version.